models.go

```diff
@@ -3,18 +3,41 @@ package outline
 import "time"
 
 type (
-	DocumentID      string
-	DocumentShareID string
-	DocumentUrlID   string
-	CollectionID    string
+	DocumentID       string
+	ParentDocumentID string
```

Comment:
Not sure if if we need this as an extra type..
Maybe `DocumentId` is enough, but naming it `parentDocumentId` 🤷